target/linux/ramips/image/mt7620.mk

```diff
@@ -244,6 +244,15 @@ define Device/u25awf-h1
 endef
 TARGET_DEVICES += u25awf-h1
 
+define Device/u35wf
+  DTS := U35WF
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := Kimax U35WF
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-mt76 kmod-usb-ohci \
+  			kmod-usb-ledtrig-usbport kmod-scsi-core kmod-usb-storage
+endef
+TARGET_DEVICES += u35wf
```

Comment:
Please use `TARGET_DEVICES += kimax_u35wf` or `TARGET_DEVICES += kimax_u35awf` depending on what you intended to do with the compatible string.

Reply:
my mistake, is supposed to be kimax_u35wf